tests/filecheck/mlir-conversion/with-mlir/dialects/llvm/llvm_pointer_ops.mlir

```diff
@@ -16,10 +16,10 @@
 // CHECK: "builtin.module"() ({
 // CHECK-NEXT:   %0 = "arith.constant"() {"value" = 0 : i64} : () -> i64
 // CHECK-NEXT:   %1 = "llvm.inttoptr"(%0) : (i64) -> !llvm.ptr<i32>
-// CHECK-NEXT:   %2 = "llvm.load"(%1) : (!llvm.ptr<i32>) -> i32
+// CHECK-NEXT:   %2 = "llvm.load"(%1) {"ordering" = 0 : i64} : (!llvm.ptr<i32>) -> i32
```

Comment:
What is this additional attribute here? Why do we not need to add it to the Load op? I guess it is just implicitly 0 if not  set?

Reply:
My guess is canonicalization?